xoinvader/level.py

```diff
@@ -1,17 +1,17 @@
-"""Module for creating and maintaining waves of incoming enemies."""
+"""Module for creating and maintaining xoinvader levels."""
 
-class EnemyWave(object):
-    """Container for enemy wave behavior.
+class Level(object):
```

Comment:
Ok.

Also please drop that habit of adding trailing ")" to your comments. It doesn't make it look friendly, it howewer does make it look much more condescending.